packages/indicators/src/__tests__/__integration/arithmeticAnalytics.fixtures.json

```diff
@@ -0,0 +1,447 @@
+        "answers": {
+          "Female": "6",
+          "Male": "8"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "IT",
+        "submission_time": "2019-06-05T15:00:00Z",
+        "answers": {
+          "Female": "10",
+          "Male": "20"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "IT",
+        "submission_time": "2020-07-07T15:00:00Z",
+        "answers": {
+          "Female": "30",
+          "Male": "40"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "ES",
+        "submission_time": "2020-01-01T15:00:00Z",
+        "answers": {
+          "Female": "100"
+        }
+      },
+      {
+        "surveyCode": "Births",
+        "entityCode": "ES",
+        "submission_time": "2020-02-02T15:00:00Z",
+        "answers": {
+          "Female": "150",
+          "Male": "200"
+        }
+      }
+    ]
+  },
+  "testCases": [
+    {
+      "description": "Throws if config is empty",
+      "input": {
+        "indicatorCodes": ["EmptyConfig"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2020-12-31",
+          "organisationUnitCodes": ["TO"]
+        }
+      },
+      "throws": true,
+      "expected": "Should not be empty"
+    },
+    {
+      "description": "Returns empty results if org unit has no data",
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2020-12-31",
+          "organisationUnitCodes": ["TO"]
+        }
+      },
+      "expected": []
+    },
+    {
+      "description": "Returns empty results if date range has no data",
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2017-01-01",
+          "endDate": "2017-12-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": []
+    },
+    {
+      "description": "Evaluates an arithmetic formula",
+      "input": {
+        "indicatorCodes": ["MonthlyBirths"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2019-01-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": [
+        {
+          "dataElement": "MonthlyBirths",
+          "organisationUnit": "GR",
+          "period": "20190101",
+          "value": 3
+        }
+      ]
+    },
+    {
+      "description": "Evaluates a boolean formula (result: true)",
+      "input": {
+        "indicatorCodes": ["FemaleGtMale"],
+        "fetchOptions": {
+          "startDate": "2019-01-01",
+          "endDate": "2019-01-31",
+          "organisationUnitCodes": ["GR"]
+        }
+      },
+      "expected": [
+        {
+          "dataElement": "FemaleGtMale",
+          "organisationUnit": "GR",
+          "period": "20190101",
+          "value": 0
```

Comment:
So `true` is `0`, and `false` `1`?

Reply:
Umm no actually this is an error in the test case description!

Perhaps following the feminist tradition in Tupaia I subconsciously wanted the `FemaleGtMale` statement to be `true` rather than false 😉

Reply:
Ah right! Haha nice, Elena would be proud